scripts/create_accessions.py

```diff
@@ -220,6 +225,48 @@ def generate_accession(facility_id: int) -> Dict:
     }
 
 
+def generate_accession_v2(facility_id: int, species_ids: List[int]) -> Dict:
+    num_collectors = randint(1, 3) if randint(0, 2) == 0 else 0
+    collectors = [generate_staff_responsible() for n in range(num_collectors)] or None
+
+    bag_numbers = generate_bag_numbers()
+    geolocations = (
+        list([generate_geolocation() for x in bag_numbers]) if bag_numbers else None
+    )
+    viability_test_types = [generate_test_type()]
+
+    species = generate_species()
+    family = generate_family(species)
```

Comment:
unused? we can skip passing species_ids in, if so.

Reply:
Quite right. Originally I had the v1 and v2 code in the same functions and I forgot to clean this up when I split them.

Reply:
Oops. I missed catching that `family` is unused.